babel.config.js

```diff
@@ -88,6 +90,14 @@ module.exports = api => {
         {
           modules,
           targets,
+          loose: true,
+          exclude: [
+            // not needed, we don't have Symbols
+            '@babel/plugin-transform-typeof-symbol',
+            // we are passing an option
+            '@babel/plugin-transform-classes',
+            '@babel/proposal-object-rest-spread',
```

Comment:
do we pass loose: true for all plugins, or specifically for these two?

I'm in favour of passing loose: true generally